Gemfile.lock

```diff
@@ -58,7 +58,7 @@ GEM
       zeitwerk (~> 2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
-    autoprefixer-rails (9.7.3)
+    autoprefixer-rails (9.7.4)
```

Comment:
### [autoprefixer-rails](https://github.com/ai/autoprefixer-rails)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 9.7.3 → 9.7.4

[_[change-log](https://github.com/ai/autoprefixer-rails/blob/master/CHANGELOG.md), [source-code](https://github.com/ai/autoprefixer-rails)_]





<details>
<summary>Commits</summary>

A change of **2** commits. See the full changes on [the compare page](https://github.com/ai/autoprefixer-rails/compare/9.7.3...9.7.4).

These are the individual commits:
- (b29ced2) [Update autoprefixer.js with fresh Can I Use](https://github.com/ai/autoprefixer-rails/commit/b29ced2b2ea75e9003b70758f2c4a261e4cd2cba)
- (dfcb465) [Release 9.7.4 version](https://github.com/ai/autoprefixer-rails/commit/dfcb4656b350e272b1fda7897824e5240a69ae54)


</details>